tests/PHPUnit/Integration/Columns/UpdaterTest.php

```diff
@@ -0,0 +1,285 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+namespace Piwik\Plugins\Test\Columns;
+
+use Piwik\Columns\Updater as ColumnsUpdater;
+use Piwik\Common;
+use Piwik\Db;
+use Piwik\DbHelper;
+use Piwik\Plugin\Dimension\ActionDimension;
+use Piwik\Plugin\Dimension\ConversionDimension;
+use Piwik\Plugin\Dimension\VisitDimension;
+use Piwik\Tests\Framework\TestCase\IntegrationTestCase;
+use Piwik\Updater;
+
+// NOTE: we can't use PHPUnit mock framework since we have to set columnName/columnType. reflection will set it, but
```

Comment:
I think they're better looking, but PHP support inner classes, so I think it usually creates clutter.